# Disables the dial registry's repeating discovery timer
# This caused unnecessary SSDP network spam

--- a/chrome/browser/media/router/discovery/dial/dial_registry.cc
+++ b/chrome/browser/media/router/discovery/dial/dial_registry.cc
@@ -196,10 +196,6 @@
 
   dial_ = CreateDialService();
   dial_->AddObserver(this);
-  DoDiscovery();
-  repeating_timer_.reset(new base::RepeatingTimer());
-  repeating_timer_->Start(FROM_HERE, refresh_interval_delta_, this,
-                          &DialRegistry::DoDiscovery);
 }
 
 void DialRegistry::DoDiscovery() {
